pkg/mobile/types.go

```diff
@@ -28,11 +32,19 @@ type DeviceMetric struct {
 	PlatformVersion string `json:"platformVersion"`
 }
 
+const clientIdMaxLength = 128
+
+var clientIdLengthError = fmt.Sprintf("clientId exceeded maximum length of %v", clientIdMaxLength)
```

Comment:
@darahayes I'm missing context but is the clientId not a fixed length across the SDKs? Since you've pinged @psturc on the PR description probably you two are aware of why this is needed.